WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Deposits Overview/WooPaymentsDepositsCurrencyOverviewView.swift

```diff
@@ -42,55 +38,46 @@ struct WooPaymentsDepositsCurrencyOverviewView: View {
                 }
 
                 if isExpanded {
+                    Text(viewModel.balanceTypeHint)
+                        .font(.footnote)
+                        .foregroundColor(.secondary)
+                        .padding(.bottom, Layout.padding)
                     Divider()
-
-                    HStack(alignment: .top) {
-                        Image(systemName: "calendar")
-                            .accessibilityHidden(true)
-                        Text(viewModel.balanceTypeHint)
-                            .font(.footnote)
-                    }
-                    .foregroundColor(.secondary)
-                    .padding(.vertical, 8)
                 }
             }
 
             if isExpanded {
-                Text(Localization.depositsHeader.localizedUppercase)
+                Text(Localization.lastDepositHeader.localizedUppercase)
                     .font(.subheadline)
                     .foregroundColor(.secondary)
                     .frame(maxWidth: .infinity, alignment: .leading)
+                    .padding(.top, Layout.padding)
                     .accessibilityAddTraits(.isHeader)
-                LazyVGrid(columns: [GridItem(.flexible(maximum: 60), alignment: .leading),
+                LazyVGrid(columns: [GridItem(.flexible(minimum: 100), alignment: .leading),
                                     GridItem(alignment: .leading),
-                                    GridItem(.flexible(minimum: 100), alignment: .leading),
                                     GridItem(alignment: .trailing)],
```

Comment:
Since the new design no longer requires a grid, I'd suggest using the `AdaptiveStack` to avoid having to guess the width of the columns and better support accessibility:

```Swift
               AdaptiveStack(horizontalAlignment: .leading, spacing: Layout.elementSpacing) {
                    HStack {
                        Image(systemName: "calendar")
                            .accessibilityHidden(true)
                        Text(viewModel.lastDepositDate)
                            .foregroundColor(.primary)
                    }
                    WooPaymentsDepositsBadge(status: viewModel.lastDepositStatus)
                    Spacer()
                    Text(viewModel.lastDepositAmount)
                        .foregroundColor(.primary)
                }
```

| Before | After |
| ----- | ----- |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/3673d19e-e7fa-4ab3-9c64-36ce8891d239" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/3dce834c-f0ae-404a-993e-31c3f35b1e68" width=320 /> |

Reply:
Much better in smaller screens indeed, updated here: https://github.com/woocommerce/woocommerce-ios/pull/11533/commits/99a1db053f299818606b94aaf5adc6d47cc86438